docs/source/miscellaneous/index.rst

```diff
@@ -7,3 +7,4 @@ Miscellaneous
    :titlesonly:
 
    contributing
+   doc/adr/index
```

Comment:
Sorry probably should have added this explicitly, but it's to help make it easier to us adr-tool which assumes they are located in doc/adr